app-dicts/wnn-ldic/wnn-ldic-1.04.ebuild

```diff
@@ -11,7 +11,6 @@ SRC_URI="mirror://gentoo/ldic-${PV}-wnn.txt"
 LICENSE="HPND"
 SLOT="0"
 KEYWORDS="~amd64 ~riscv ~x86"
-IUSE=""
 
 DEPEND="app-i18n/freewnn"
```

Comment:
Check if this is really an RDEPEND too.

Reply:
Also, maybe we should change it to >=app-i18n/freewnn-1.1.1_alpha23 and then we can drop the has_version?

Reply:
Will do, do you check those with running tests?

Reply:
You'll have to look at what freewnn is and grep the codebase of this package to see how it really uses it, and also look at ELF NEEDED of the installed files (but obviously iin this case it's a dictionary so maybe it really is just DEPEND for building it).